Client/Launcher/premake5.lua

```diff
@@ -13,9 +13,14 @@ project "Launcher"
 
     links {
         "Utility",
-        "dbghelp", "psapi", "comctl32", "wininet", "winhttp", "Shlwapi"
+        "dbghelp", "psapi", "comctl32", "wininet", "winhttp", "Shlwapi",
+        "PEFramework"
     }
-
+    
+    includedirs {
+        "../../Vendor/eirrepo/"
```

Comment:
It requires rwlist.hpp, MemoryRaw.h, MemoryUtils.h and MemoryUtils.stream.h . Those are being created by PEFramework.